src/ui/SolidityTypeFormatter.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
 function formatData (key, data) {
   var style = fontColor(data)
   var keyStyle = data.isProperty ? 'color:#847979' : ''
+  if (data.type === 'string') {
+    data.self = '"' + data.self + '"'
```

Comment:
Hm, we could even use `Json.stringify` here, then it would also escape quotes inside the string and it can be copy-pasted easier.